src/manifest.json

```diff
@@ -13,8 +13,15 @@
 	},
 	"permissions": [
 		"storage",
-		"clipboardWrite"
+		"clipboardWrite",
+		"contextMenus",
```

Comment:
Why are we requesting this permission? This is for right-click context menus, not the browser action.

Reply:
I use this API, it's irrelevant whether the right click is on the document or on the browser action: https://developer.chrome.com/apps/contextMenus

Either way it's just a formality, Chrome doesn't display that permission (just like `storage`)